mypy.ini

```diff
@@ -0,0 +1,10 @@
+[mypy]
+pretty = true
+show_column_numbers = true
+show_error_codes = true
+warn_no_return = true
+warn_redundant_casts = true
+warn_unused_configs = true
+warn_unused_ignores = true
+ignore_missing_imports = true
```

Comment:
This will need to be addressed (maybe in a separate PR)

Reply:
Makes sense